client/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <title>Journal</title>
+    <script defer src="front.js"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1" /> 
+    <link href="https://fonts.googleapis.com/css2?family=Inconsolata&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <form class="flex-container">
+        <section id="inputs">
+            <input type="text" placeholder=">> Title..." class="title" maxlength="100" required>
```

Comment:
really nice use of max length, to ensure people do not go over the set varchar limits. 
if there was not this, I would have said to maybe implement an error message to state they have gone over the character limit, I think it would work similar to the character counter from the previous group project! however, this is very clever as it physically prevents the user from inputting extra characters :)